home.md

```diff
@@ -0,0 +1,75 @@
+<img width="500" align="right" alt="illustration" src="https://user-images.githubusercontent.com/82679007/150655220-d023b26e-ceab-4b37-9d66-0ad6b20f0628.png">
+
+# Mentorship...for you!
+
+### Do you need guidance on a project? Need help learning how to contribute to open source? Guidance in settling into the Galaxy ecosystem? More?
```

Comment:
Is it a normal paragraph? Then you can remove the `###` before the line

Reply:
I am not sure to understand what you mean here

Reply:
Like a landing page hero section, there's usually a big headline and a small copy just beneath it and above the CTA button. the font size is quite bigger than the font size of the normal body text(paragraph). that's why I used ### and not the bold syntax

Reply:
I get now what you mean. But it will probably not work directly like this, specially if you have an image before. You will probably need to define metadata on the top of the file with title, subtitle. Something like:

```
---
title: Your title
subtitle: Your subtitle
---
```